### PR TITLE
Init

### PR DESCRIPTION
**Pull request by kucabix**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,133 @@
...
+  toggleCurrency = () => {
+    this.setState({
+      fromCurrency: this.state.toCurrency,
+      toCurrency: this.state.fromCurrency
+    })
+  }
+
+  render() {
+    const {fromCurrency, toCurrency, exchangeRate, fromCountry, toCountry, isLoading, selectedCurrency} = this.state
+    const loadingClass = isLoading ? '-loading':''
```
Comment (tomeroth) [inline src/App.js]: put spaces before and after `:`

File: src/App.js [hunk 2]
```diff
@@ -1,26 +1,133 @@
...
+    const selectClass = selectedCurrency.length ? '':'-none'
```
Comment (tomeroth) [inline src/App.js]: same as above

File: src/Currencies.js [hunk 3]
```diff
@@ -0,0 +1,150 @@
+const currencies = [
+  {value:"AED", label: "AED | United Arab Emirates Dirham"},
```
Comment (tomeroth) [inline src/Currencies.js]: code formatting, there should me more spaces like
`{ value: "AED", label: "AED | United Arab Emirates Dirham" },`

File: src/App.js (hunk 1 above)
Comment (kucabix) [reply src/App.js]: Done

File: src/App.js (hunk 2 above)
Comment (kucabix) [reply src/App.js]: Done

File: src/Currencies.js (hunk 3 above)
Comment (kucabix) [reply src/Currencies.js]: Done

**Pull request merged**